src/App.ts

```diff
@@ -1,4 +1,12 @@
 import { Server } from './Server';
+import { loadConfig } from './utils/ConfigLoader';
+
+/**
```

Comment:
I do not find these comments useful. Code should be self explanatory.

Reply:
We could rename function to `loadEnvironmentVariablesFromFiles()` - not so short but does not require comment IMO.